### ISSUE TITLE
Strawberry Seeds cannot be composted

### ISSUE THREAD

**Issue by BrekiTomasson** [Motschen/SimplyStrawberries#1]
Unlike most other seeds in the game, Strawberry Seeds cannot be put into a composter. Not sure if this is intentional on your part or an oversight, but thought it'd be best to let you know.

**Comment by Motschen** [comment]
Yes, it's an oversite. In modern minecraft there're just so many things a mod creator has to think of...
Will probably be corrected in the next update.
